src/v4/injectGlobalToCreateGlobalStyle.js

```diff
@@ -0,0 +1,125 @@
+"use strict";
+
+const t = require("babel-core").types;
+const { prop, set, has, flow, findIndex } = require("lodash/fp");
+const { manipulateOptions } = require("./common");
+
+const WARNING =
+  "An injectGlobal usage was converted to createGlobalStyles via codemod but needs to be hooked up. See styled-components.com/docs/api#createglobalstyle for instructions.";
```

Comment:
Can you make this a full URL with https etc?

Reply:
Done.